docs/src/usage.md

```diff
@@ -150,17 +150,20 @@ Since the release of morecantile `1.3.1`, users can automatically extend morecan
 
     - Filename HAVE TO be the same as the TMS identifer
     - Filename HAVE TO be *without special characters* `[a-zA-Z0-9_]`
+    
+### Export TMS definiton to JSON dictionary
 
-## Morecantile + Pydantic
-
-Morecantile uses [Pydantic](https://pydantic-docs.helpmanual.io) to define and validate TileMatrixSet documents.
+```
+import json
+import morecantile
 
-From Pydantic docs:
-> Define how data should be in pure, canonical python; validate it with pydantic.
+tms = morecantile.tms.get("WebMercatorQuad")
+tms_json = json.loads(tms.json()) # the .json() func does not return JSON dict directly
```

Comment:
> the .json() func does not return JSON dict directly

what do you mean by this? if you want a dict you can use `.dict()`, `.json()` will return a JSON string

Reply:
Ok, but this must be also clarified

Reply:
what is `this` ?

Reply:
well that's why we mentioned that morecantile is built on top of Pydantic, so we assume user will either know pydantic (which is a pretty well known library) or at least go check its documentation so they understand what methods the model implements